infrastructure/nomad/playbooks/deploy.yml

```diff
@@ -495,7 +495,7 @@
             echo "${ALLOC}" | jq --arg address "0x${ADDRESS}" '
```

Comment:
Can we remove the for-loop here to enforce that someone doesn't accidentally add another allocation?

Reply:
Done.